yt_astro_analysis/utilities/testing.py

```diff
@@ -14,33 +14,26 @@
 # -----------------------------------------------------------------------------
 
 import os
-import shutil
-import tempfile
-from unittest import TestCase
+import warnings
 
+import yt
 from yt.config import ytcfg
-from yt.data_objects.time_series import SimulationTimeSeries
-from yt.loaders import load_simulation
-from yt.utilities.answer_testing.framework import AnswerTestingTest
 
 
-class TempDirTest(TestCase):
-    """
-    A test class that runs in a temporary directory and
-    removes it afterward.
-    """
-
-    def setUp(self):
-        self.curdir = os.getcwd()
-        self.tmpdir = tempfile.mkdtemp()
-        os.chdir(self.tmpdir)
-
-    def tearDown(self):
-        os.chdir(self.curdir)
-        shutil.rmtree(self.tmpdir)
+def data_dir_load(fn, *args, **kwargs):
+    # wrap yt.load but only load from test_data_dir
+    path = os.path.join(ytcfg.get("yt", "test_data_dir"), fn)
+    return yt.load(path, *args, **kwargs)
 
 
 def requires_sim(sim_fn, sim_type, file_check=False):
```

Comment:
If this is being deprecated, what functionality is replacing it? Same for `can_run_sim`

Reply:
None. The idea is that *if* anyone is using those functions downstream, they should be able to vendor them because they're very simple.